docs/source/sysadmin_guide/about_dynamic_consensus.rst

```diff
@@ -0,0 +1,247 @@
+.. _about-dynamic-consensus-label:
+
+***********************
+About Dynamic Consensus
+***********************
+
+Sawtooth :term:`dynamic consensus` lets you choose from a variety of consensus
+algorithms for the network, including PBFT, PoET, Raft, and Devmode.
+
+* For a new network, the administrator for the first node specifies the
+  consensus settings in the :term:`genesis block`. All administrators must
+  must install and start a consensus engine (the same version) and any
+  transaction processors required for consensus (such as PoET Validator
+  Registry).
+  See :doc:`/sysadmin_guide/setting_up_sawtooth_poet-sim` for this procedure.
+
+* To change consensus for a running network, administrators must coordinate the
+  changes for all nodes. Each administrator must install and start the new
+  consensus engine. Then one administrator submits a transaction to change the
+  on-chain consensus settings.
+
+  After the consensus has changed, each administrator can stop the previous
```

Comment:
It is worth mentioning that if a node is added or is catching up, they would need to have the correct consensus engines/tp connected to handle all the previous versions of consensus used to build the chain. For example if the first half of the chain used  x and then switched to y a new node would need x and y until it caught up to the other nodes.

Reply:
💡

Reply:
Would it make sense to tell admins to just keep the old engines/TPs running in case a node is added later?  (And, of course for slow nodes who aren't caught up yet.)

Reply:
Well technically, the node that's caching up would only need x until it reaches the block where the network switched to y. To answer your question, @chenette, existing nodes won't have to keep old stuff running; when a new node is added, it will have to run the old engine + TPs.